services/volume-inspector/api/api.go

```diff
@@ -0,0 +1,107 @@
+package api
+
+import (
+	"net/http"
+	"github.com/gorilla/mux"
+	"time"
+	"io/ioutil"
+	"encoding/json"
+	"os"
+	"log"
+)
+
+// Volume folder content
+type VolumeItem struct {
+	Name       string `json:"name"`
+	Size	   int64 `json:"size"`
+	Modified   time.Time `json:"modified"`
+	IsFolder   bool `json:"isFolder"`
+	FolderTree VolumeItems `json:"folderTree"`
+}
+
+type VolumeItems []VolumeItem
```

Comment:
Please never create two variables that differ by only one letter. One common solution is to rename this one to VolumeItemList. I'm also not sure that this type is needed, you can use []VolumeItem everywhere.